src/data-access/MwApiLexemeCreator.ts

```diff
@@ -37,10 +45,27 @@ export default class MwApiLexemeCreator implements LexemeCreator {
 				language: lexemeLanguageItemId,
 				lexicalCategory: lexicalCategoryItemId,
 			} ),
+			errorformat: 'html',
+			formatversion: 2,
```

Comment:
Aren't these duplicating what we have in init.ts, the `new mw.API()` constructor?

Reply:
Yes, but I thought they should be included here again, since we now rely on both of these parameters – if the code relies on a certain API format, it should also ensure that it’s actually in use, not just hope that the given `api` instance does the right thing.

I could be convinced otherwise, but then the requirements should at least be documented in the class comment, I think.

Reply:
> Yes, but I thought they should be included here again, since we now rely on both of these parameters

Yes, that is a good point. But shouldn't we, by that logic, also include the `uselang` here? We are also depending on it being returned in a specific language.

Reply:
I wouldn’t say so – the code here works fine with any `uselang`, whether it’s appropriate to show to users is a different question (I think we don’t even have the language code in question in this class?) – whereas `errorformat` and `formatversion` change the JSON structure: without `errorformat`, there would be a single `error` instead of an `errors` array, and without `formatversion` the `html` key would be `*` instead.